packages/core-js/utils/ast/finders.js

```diff
@@ -82,19 +84,30 @@ function findYulCaseValues(contractName, ast) {
   return items;
 }
 
-function findFunctionSelectors(contractName, asts) {
+function _findFunctionSelectors(contractNode) {
   const selectors = [];
-  const contractNode = asts[contractName];
-  if (!contractNode) {
-    return selectors;
-  }
 
   for (const functionDefinition of findAll('FunctionDefinition', contractNode)) {
-    selectors.push({ selector: '0x' + functionDefinition.functionSelector });
+    if (functionDefinition.functionSelector) {
```

Comment:
this if is to exclude FunctionDefinitions that doesn't have a selector (i.e. internal functions). Previously we were adding lots of '0xundefined' to the array.